### ISSUE TITLE
did not find method - onController()

### ISSUE THREAD

**Issue by moz4r** [MyRobotLab/myrobotlab#79]
Hi !

Trying to kill errors in the black box, I like clean black box ! I dont understand this one

11:50:27.660 [SwingGui] ERROR c.myrobotlab.framework.Service - did not find method - onController()

this caused if I start a servo

An idea ?

**Comment by supertick** [comment]
Thanks moz4r .. Ya, I can see the errors.  It has a couple problems.  The controllers nor the control was updating the gui.  Good find !  I'm fixing now

**Comment by supertick** [comment]
This was a bit messy.  There was onRefreshControllers and onController and onState.  These are all callbacks which potentially refresh the state of the gui.  onState is the "big" one and its logic was correct.  Sometimes we choose to decompose the methods into more discrete callbacks like onController which would just refresh the Servo's controller state.  In this case it was a bad idea - so I removed it and fixed up onRefreshControllers - there was a multitude of fixes ..

Tested - attaching a controller, detaching a controller,  adding a new controller to the system (it appeared in the dropdown) ..  no ERRORs .. looks good.
